perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -32,10 +32,16 @@ sub begin : Private {
 
     $c->uri_disposition('relative');
 
-    if ( $c->cobrand->moniker eq 'zurich' || $c->cobrand->moniker eq 'seesomething' ) {
-        $c->detach( '/auth/redirect' ) unless $c->user_exists;
-        $c->detach( '/auth/redirect' ) unless $c->user->from_body;
+    # Must be logged in to see the admin, as a body user or a super user
+    unless ( $c->user_exists && ( $c->user->from_body || $c->user->is_superuser )) {
```

Comment:
I think the default should perhaps (for now at least) be superuser only, with the cobrand allowing more if it wants (can look again once permissions are all up and set up I guess, and an admin interface for administering them exists :) ).